src/SetHandler.cpp

```diff
@@ -228,7 +228,7 @@ void SetHandler::writeTab(const string& dataFN, SanityCheck * pCheck) {
 
 void SetHandler::readPSMs(istream& dataStream, std::string& psmLine, 
     bool hasInitialValueRow, bool& concatenatedSearch,
-    std::vector<OptionalField>& optionalFields) {
+    std::vector<OptionalField>& optionalFields, std::string decoyPrefix) {
```

Comment:
Can we avoid passing decoyPrefix to all kinds of functions by making it a member variable?

Reply:
Fixed in commit b2cdb10